app/views/ReportingFrequencyView.scala.html

```diff
@@ -179,9 +213,19 @@
     @h1(msg = messages("reporting.frequency.h1"), id = Some("reporting-frequency-heading"))
     @ceasedBusinessWarning
     @itsaStatusTable
-    @if(viewModel.isAnyOfBusinessLatent) {
-        @detailsDropdown("reporting.frequency.latency.details.section.header", latentBusinessText)
+
+
+    @if(optInOptOutContentUpdateR17) {
+        @if(viewModel.isAnyOfBusinessLatent) {
+            @detailsDropdown("reporting.frequency.latency.details.section.header.updateR17", dropDownContentOptInOptOutContentUpdateR17, Some("latency-details-dropdown"))
+        }
+    } else {
+        @if(viewModel.isAnyOfBusinessLatent) {
+            @detailsDropdown("reporting.frequency.latency.details.section.header", latentBusinessDropdownContent, Some("latency-details-dropdown"))
+        }
     }
```

Comment:
Instead of having it check the R17 content first it'll be better to have it wrapped around a latent business check so it isn't calling the same if statement twice.

```suggestion

@if(viewModel.isAnyOfBusinessLatent) {
    @if(optInOptOutContentUpdateR17) {
        @detailsDropdown("reporting.frequency.latency.details.section.header.updateR17", dropDownContentOptInOptOutContentUpdateR17, Some("latency-details-dropdown"))
    } else {
        @detailsDropdown("reporting.frequency.latency.details.section.header", latentBusinessDropdownContent, Some("latency-details-dropdown"))
    }
}
```